pkg/audit/audit.go

```diff
@@ -67,15 +67,15 @@ func (a *Audit) Log(name, op, value string) {
 	}
 	jsonEntry = append(jsonEntry, '\n')
 	if _, err := a.file.Write(jsonEntry); err != nil {
-		klog.Fatalf("log failed with error: %v", err)
+		klog.Fatalf("log failed with error, err: %v", err)
```

Comment:
Fatalf may lead to the abrupt termination in case of error. 
We can modify the severity.

Reply:
Done. Modified the severity if its not expected.